Finish Lesson 1.3

StuvaScript opened the pull request:
Create TodoListItem component.
Create an "addTodo" handler.
Utilize the useState hook to store new todo data.

StuvaScript committed:
Finish Lesson 1.3

Create TodoListItem component.
Create an "addTodo" handler.
Utilize the useState hook to store new todo data.

StuvaScript committed:
Turn off prop-type checker

BoyanLiuu commented on src/AddTodoForm.jsx:
```diff
@@ -1,8 +1,16 @@
-export default function AddTodoForm() {
+export default function AddTodoForm({ onAddTodo }) {
```
The following code would be better:
```
const handleAddTodo = (event) => {
    event.preventDefault();
    const todoTitle = event.target.title.value;
    console.log(todoTitle);
    onAddTodo(todoTitle);
    event.target.title.value = "";
};

```

We converted the function declaration to an arrow function and replaced let with const for the todoTitle variable since its value doesn't change. This makes the code more concise and leverages ES6 syntax.

StuvaScript committed:
Changes in AddTodoForm file

Changed a "let" variable into a "const" variable.

Changed a function declaration into a function
expression.
